From ed3195ac4e24cae4bdae4ae5f0fcc0a68612b203 Mon Sep 17 00:00:00 2001
From: Etaash Mathamsetty <etaash.mathamsetty@gmail.com>
Date: Wed, 30 Jul 2025 18:17:41 -0400
Subject: [PATCH 197/506] ntdll: Add FIXME to SystemModuleInformation.

---
 dlls/ntdll/unix/system.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/dlls/ntdll/unix/system.c b/dlls/ntdll/unix/system.c
index a24c2600f81..4b3eab66410 100644
--- a/dlls/ntdll/unix/system.c
+++ b/dlls/ntdll/unix/system.c
@@ -3502,6 +3502,8 @@ NTSTATUS WINAPI NtQuerySystemInformation( SYSTEM_INFORMATION_CLASS class,
         ULONG i;
         RTL_PROCESS_MODULES *smi = info;
 
+        FIXME("semi-stub!\n");
+
         len = offsetof( RTL_PROCESS_MODULES, Modules[ARRAY_SIZE(fake_modules)] );
         if (len <= size)
         {
@@ -3837,6 +3839,8 @@ NTSTATUS WINAPI NtQuerySystemInformation( SYSTEM_INFORMATION_CLASS class,
         ULONG i;
         RTL_PROCESS_MODULE_INFORMATION_EX *module_info = info;
 
+        FIXME("semi-stub!\n");
+
         len = sizeof(*module_info) * ARRAY_SIZE(fake_modules) + sizeof(module_info->NextOffset);
         if (len <= size)
         {
-- 
2.53.0

